apps/content-publishing-api/src/controllers/v1/asset.controller.v1.ts

```diff
@@ -39,7 +39,6 @@ export class AssetControllerV1 {
         .addFileTypeValidator({
           fileType: DSNP_VALID_MIME_TYPES,
         })
-        // TODO: add a validator to check overall uploaded size
```

Comment:
we don't need this since we limited the number of files and the size of each file and we also have timeout policices defined.